### PR TITLE
Shooter rewrite merge back to main

### PR DESCRIPTION
**Pull request by Spenceroni-Pepperoni**

### CODE REVIEW

**Review by Gliu1211 — COMMENTED** [review]
I think there are unresolved merge conflicts
Edit: resolved

**Comment by Spenceroni-Pepperoni** [comment]
> I think there are unresolved merge conflicts

Fixed Merge conflicts

**Review by Gliu1211 — APPROVED** [review]
looks good

File: src/main/java/frc/robot/commands/Toggles/ToggleIntakeDirct.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+
```
Comment (liam-a-wms) [inline src/main/java/frc/robot/commands/Toggles/ToggleIntakeDirct.java]: If this command doesn't do anything, why is it here?

File: src/main/java/frc/robot/commands/Toggles/ToggleSpinIn.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {}
+
```
Comment (liam-a-wms) [inline src/main/java/frc/robot/commands/Toggles/ToggleSpinIn.java]: If this command doesn't do anything, why is it here?

File: src/main/java/frc/robot/commands/Toggles/ToggleSpinOut.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+
```
Comment (liam-a-wms) [inline src/main/java/frc/robot/commands/Toggles/ToggleSpinOut.java]: If this command doesn't do anything, why is it here?

**Review by liam-a-wms — CHANGES_REQUESTED** [review]

**Review by SixtyFiveOhTwo — APPROVED** [review]
looks good

File: src/main/java/frc/robot/RobotContainer.java [hunk 4]
```diff
@@ -132,17 +136,24 @@ public RobotContainer() {
...
 
         //shooter controls
 
         //intake Controls
         leftShoulder.whileHeld(new IntakeSpin(m_intake, 0.75));
+        rightShoulder.whileHeld(new IntakeSpin(m_intake, -0.75));
         bButton.whenPressed(new InstantCommand(() -> m_intake.pistonToggle()));
+        xButton.whileHeld(new teleopCanal(canal));
 
         button7.toggleWhenPressed(new ShooterSpin(shoot, twiststick, 0.50));
```
Comment (Gliu1211) [inline src/main/java/frc/robot/RobotContainer.java:161]: What is button7?

**Review by Gliu1211 — DISMISSED** [review]

**Review by liam-a-wms — APPROVED** [review]

**Pull request merged**